include/net/net_context.h

```diff
@@ -45,21 +45,30 @@ enum net_context_state {
 	NET_CONTEXT_LISTENING = 3,
 };
 
+enum {
```

Comment:
Too many things in that patch. I'd slice in incremental patches: first context, then core, then ethernet l2.

Reply:
I split as many as I can in new version.